docs/_docs/user-interface/simplify.md

```diff
@@ -0,0 +1,28 @@
+---
+title: Geographical simplification
+permalink: /docs/simplify/
+---
+
+### Geographical simplification
+
+Use this step in case you want to simplify heavy geographical data.
+
+When simplifying your data, every point that is closer than a specific distance to the previous one is suppressed.
+This step can be useful if you have a very precise shape for a country (such as one-meter precision), but want to quickly
+draw a map chart. In that case, you may want to *simplify* your data.
```

Comment:
Not easy to explain! Maybe we can link to https://geopandas.org/en/stable/docs/reference/api/geopandas.GeoSeries.simplify.html ?

Reply:
I was hesitating, but it seems complex for user docs... Maybe I can move this paragraph to the tech docs and just add a simpler explanation here ? Something like _This step reduces the size of your data by making it less precise. The higher you set the tolerance, the smaller (and the less precise) your data will be. See the  **technical documentation** for details_

Reply:
Yes, it's simpler!